components/board/genericlinux/board.go

```diff
@@ -66,47 +81,49 @@ func newBoard(
 		b.gpios[fmt.Sprintf("%d", pinNumber)] = b.createGpioPin(mapping)
 	}
 
-	if err := b.Reconfigure(ctx, nil, conf); err != nil {
+	if err := b.ReconfigureParsedConfig(ctx, conf); err != nil {
```

Comment:
this removes the reconfigure functionality from the creator. Revert it to reconfigure, and keep reconfigure parsed config where it is currently.

Reply:
this function doesn't have access to a resource.Config object though